index.src.html

```diff
@@ -1229,6 +1242,74 @@ <h2 id="sample-reports">Sample Reports</h2>
   </div>
 </section>
 
+<section>
+
+</section>
+  <h2 id="automation">Automation</h2>
+
+  For the purposes of user-agent automation and application testing, this
+  document defines a number of <a>extension commands</a> for the [[WebDriver]]
+  specification.
+
+  <h3 id="generate-test-report-command">Generate Test Report</h3>
+
+  The <dfn>Generate Test Report</dfn> <a>extension command</a> simulates the
+  generation of a <a>report</a> for the purposes of testing. This report will be
+  routed through any defined reporting <a>endpoints</a> and observed by
+  any <a>registered</a> <a>reporting observers</a>, as per normal.
+
+  The <a>extension command</a> is defined as follows:
+
+  <table>
+    <tbody>
+      <tr>
+        <th>HTTP Method</th>
+        <th><a lt="extension command prefix">Prefix</a></th>
+        <th><a lt="extension command name">Name</a></th>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{session id}/reporting</td>
+        <td>generate_test_report</td>
```

Comment:
Super nit: can you wrap the prefix and name in backticks / `tt` to make them monospace?

Reply:
Done!